compiler/semstmts.nim

```diff
@@ -229,11 +229,10 @@ proc semTry(c: PContext, n: PNode; flags: TExprFlags): PNode =
         # Overwrite symbol in AST with the symbol in the symbol table.
         a[0][2] = newSymNode(symbol, a[0][2].info)
 
-      else:
+      elif a.len == 1:
```

Comment:
What I meant was something like this:
```nim
...
elif a.len == 1:
  inc catchAllExcepts
else:
  var is_native, is_imported: bool
  ...
```

You want to run `semExceptBranchType` on the node sons if its length is greater than 1

Reply:
Got it. I misunderstood what was going on. I used more debug commands like you showed and have a better idea of what the node tree structure looks like now.